src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+    @Override
+    public List<Machine> get() {
+        return List.of(new Bulldozer(), new Bulldozer(), new Bulldozer());
```

Comment:
let's add some data to the constructor of Bulldozer
the same with other objects

Reply:
done